src/foremast/utils/vpc.py

```diff
@@ -41,7 +41,7 @@ def get_vpc_id(account, region):
             configured.
 
     """
-    url = '{0}/vpcs'.format(API_URL)
+    url = '{0}/networks/aws'.format(API_URL)
```

Comment:
The old url was deprecated while digging:
https://github.com/spinnaker/gate/blame/a3f2416a330993878b715eaba96e50f11108fd7a/gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/aws/InfrastructureService.groovy#L60